src/components/miscellaneous/ShareButton.tsx

```diff
@@ -0,0 +1,40 @@
+// Material UI Components
+import Button from '@material-ui/core/Button';
+
+import useShare from 'use-share';
+
+export type ShareProps = {
+  title?: string;
```

Comment:
Title bør være required
```suggestion
  title: string;
```